BUILDING.md

```diff
@@ -19,12 +19,20 @@ GMT is dependent on some software and libraries to run.
 Please refer to the [GMT wiki page](https://github.com/GenericMappingTools/gmt/wiki)
 for instructions to install these dependencies on various operation systems.
 
+### Required dependencies
+
 To build GMT, you have to install:
```

Comment:
Is it worth listing [Ninja](https://ninja-build.org/) as an optional dependency? I see it mentioned sometimes as a faster way to build GMT.

Reply:
Yes, now included.